generated/nidcpower/session.py

```diff
@@ -3486,6 +3486,8 @@ def set_sequence(self, values, source_delays):
         values_ctype = get_ctypes_pointer_for_buffer(value=values, library_type=_visatype.ViReal64)  # case B550
         source_delays_ctype = get_ctypes_pointer_for_buffer(value=source_delays, library_type=_visatype.ViReal64)  # case B550
         size_ctype = _visatype.ViUInt32(0 if values is None else len(values))  # case S160
+        if source_delays is not None and len(source_delays) != len(values):  # case S160
+            raise ValueError("source_delays length not equal to values length")  # case S160
```

Comment:
I wonder if it's nicer for the string to be "in python" rather than paraphrasing the python above it.
Something like:

``` python
raise ValueError("len(source_delays) != len(values)")  
```

Is that useful? Is paraphrasing useful? Or should we give a more digested message...
"Length of source_delays and values parameters don't match."